app/konnect-platform/network.md

```diff
@@ -159,3 +159,32 @@ rows:
 ## Mesh Manager hostnames
 
 If you use [Mesh Manager](/mesh-manager/) to manage your Kong service mesh, you must add the `{geo}.mesh.sync.konghq.com:443` hostname to your firewall allowlist. The geo can be `au`, `eu`, `us`, or `global`.
+
+## Specify IP addresses that can connect to {{site.konnect_short_name}}
+
+Org Admins can specify an IP address or a range of IP addresses that can connect to {{site.konnect_short_name}} on any of its channels: UI, [APIs](/konnect-api/) (including [Admin API](/admin-api/)), [decK](/deck/), and [Terraform](/terraform/). This IP range restriction applies to all {{site.konnect_short_name}} communication including:
+* Admin API
+* Control plane and data plane communication
+* {{site.konnect_short_name}} and {{site.mesh_product_name}} communication
+* {{site.konnect_short_name}} and Consumer communication
```

Comment:
This is not true yet. The current milestone only enables IP source restriction for admin API. The other three will come in future milestones.